website/docs/src/content/cloud/local-development.mdx

```diff
@@ -0,0 +1,184 @@
+---
+title: Local development
+description: Run GenSX workflows locally with the dev server
+---
+
+# Local development server
+
+GenSX provides local development experience that mirrors the cloud environment, making it easy to build and test workflows on your machine before deploying them.
+
+## Starting the dev server
+
+The `gensx start` command launches a local development server with hot-reloading:
+
+```bash
+gensx start ./src/workflows.tsx
+🔍 Starting GenSX Dev Server...
+ℹ Using project name from gensx.yaml: support-tools
+ℹ Starting development server...
+✔ Compilation completed
+✔ Generating schema
+Importing compiled JavaScript file: /Users/evan/code/gensx-console/samples/support-tools/dist/src/workflows.js
+
+
+🚀 GenSX Dev Server running at http://localhost:1337
+🧪 Swagger UI available at http://localhost:1337/swagger-ui
+
+📋 Available workflows:
+- RAGWorkflow: http://localhost:1337/org/gensx/projects/support-tools/workflows/RAGWorkflow
+- AnalyzeDiscordWorkflow: http://localhost:1337/org/gensx/projects/support-tools/workflows/AnalyzeDiscordWorkflow
+- TextToSQLWorkflow: http://localhost:1337/org/gensx/projects/support-tools/workflows/TextToSQLWorkflow
+- ChatAgent: http://localhost:1337/org/gensx/projects/support-tools/workflows/ChatAgent
+
+✅ Server is running. Press Ctrl+C to stop.
+```
+
+## Development server features
+
+### Identical API shape
+
+The local API endpoints match exactly what you'll get in production, making it easy to test your workflows before deploying them:
+
+```
+http://localhost:1337/local/{project}/default/workflows/{workflow}/run
+```
+
+Every workflow you export is automatically available as an API endpoint.
+
+### Hot reloading
+
+The development server watches your TypeScript files and automatically:
+
+1. Recompiles when files change
+2. Regenerates API schemas
+3. Restarts the server with your updated code
+
+This enables a fast development cycle without manual restarts.
+
+### API documentation
+
+The development server includes a built-in Swagger UI for exploring and testing your workflows:
+
+```
+http://localhost:1337/swagger-ui
+```
+
+![Swagger UI for local development](/cloud/local-swagger-ui.png)
+
+The Swagger interface provides:
+
+- Complete documentation of all your workflow APIs
+- Interactive testing
+- Request/response examples
+- Schema information
+
+## Local storage options
```

Comment:
I'd recommend moving this below "running workflows locally" and maybe below "configuration" too. It's non-essential information whereas the other stuff is more important

Reply:
could even take it out altogether tbh